addons/controllers/testdata/test-cluster-bootstrap-2.yaml

```diff
@@ -28,47 +20,195 @@ spec:
       cidrBlocks: [ "192.168.0.0/16","fd00:100:96::/48" ]
     services:
       cidrBlocks: [ "192.168.0.0/16","fd00:100:96::/48" ]
-  topology:
-    class: test-clusterclass-tcbt
-    version: v1.21.2---vmware.1-tkg.1
 ---
-apiVersion: run.tanzu.vmware.com/v1alpha3
-kind: ClusterBootstrapTemplate
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
 metadata:
-  name: v1.21.2---vmware.1-tkg.1
-  namespace: tkg-system
+  name: foobar.example.com.1.17.2
+  namespace: cluster-namespace-2
 spec:
-  cnis:
-    - refName: "calico.tanzu.vmware.com.0.3.0--vmware.1-tkg.1-rc.1"
-      valuesFrom:
-        providerRef:
-          apiGroup: cni.tanzu.vmware.com
-          kind: CalicoConfig
-          name: test-cluster-tcbt-2
-    - refName: "antrea.tanzu.vmware.com.21.2.0--vmware.1-tkg.1-rc.1"
-      valuesFrom:
-        providerRef:
-          apiGroup: cni.tanzu.vmware.com
-          kind: AntreaConfig
-          name: test-cluster-tcbt-2
-
+  refName: foobar.example.com
+  version: 1.17.2
+  releasedAt: "2021-05-13T18:00:00Z"
+  releaseNotes: foobar 1.17.2
+  capacityRequirementsDescription: Varies significantly based on cluster size. This should be tuned based on observed usage.
+  valuesSchema:
+    openAPIv3:
+      title: foobar.example.com.1.17.2+vmware.1-tkg.1 values schema
+      properties:
+        namespace:
+          type: string
+          description: The namespace in which to deploy fluent-bit.
+          default: tanzu-system-logging
+  licenses:
+    - 'VMware''s End User License Agreement (Underlying OSS license: Apache License 2.0)'
+  template:
+    spec:
+      fetch:
+        - imgpkgBundle:
+            image: projects-stg.registry.vmware.com/tkg/tkgextensions-dev/fluent-bit:v1.7.5_vmware.1-tkg.1
+      template:
+        - ytt:
+            paths:
+              - config/
+            ignoreUnknownComments: true
+        - kbld:
+            paths:
+              - '-'
+              - .imgpkg/images.yml
+      deploy:
+        - kapp:
+            rawOptions:
+              - --wait-timeout=5m
+              - --kube-api-qps=20
+              - --kube-api-burst=30
 ---
-apiVersion: cni.tanzu.vmware.com/v1alpha1
-kind: CalicoConfig
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
 metadata:
```

Comment:
Why these are removed? These were to test CNI selecltion

Reply:
that is because when envtest get set up for clsuterbootstrap_controller_test.go, test-cluster-bootstrap-1.yaml has already contains those resources and get deployed. No need to duplicate them here.